assets/css/styles.css

```diff
@@ -157,33 +159,143 @@ button:active, #add-button:active, #search-button:active {
 }
 
 /* recipe CARD STYLING */
+#search-results {
+    display: grid;
+    grid-template-columns: repeat(auto-fill, minmax(250px, 1fr));
```

Comment:
the line that is giving us the "Expected RBRACE at line 164, col 60." error in codacy... I can't see any missing braces